src/transformers/models/auto/modeling_auto.py

```diff
@@ -157,6 +157,7 @@
         ("jetmoe", "JetMoeModel"),
         ("jukebox", "JukeboxModel"),
         ("kosmos-2", "Kosmos2Model"),
+        ("kosmos-2.5", "Kosmos2_5Model"),
```

Comment:
Is it ok to have a "." in the model name? for other models we have "_", e.g. "qwen2_5_vl"

Reply:
`kosmos-2.5` is not in the model name or model file name: it is the `model_type`. It's OK I believe although it seems the unique one with `.`